packages/aws-cdk/lib/cli.ts

```diff
@@ -214,21 +216,25 @@ export async function exec(args: string[], synthesizer?: Synthesizer): Promise<n
         });
 
       case 'docs':
+      case 'doc':
```

Comment:
no way we just didn't honor `doc` until just now

Reply:
We did not. Maybe I should leave it out 🙈